WireUI/Sources/WireIndividualToTeamMigrationUI/IndividualToTeamMigrationViewController.swift

```diff
@@ -123,6 +123,19 @@ public class IndividualToTeamMigrationViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
     }
 
+    public convenience init(
+        useCase: any IndividualToTeamMigrationUseCase,
+        userProfileName: String,
+        actionCallback: @escaping @Sendable (Action) -> Void
+    ) {
+        self.init(
+            features: .features,
```

Comment:
Can the features change? Is this list something that needs to be injected or could it be hardcoded?

Reply:
I have no idea. All I have is Figma, and the copy presenting a feature's value prop can reasonably be expected to evolve at least a few times in the feature's first few weeks or months.